src/server/snapshot/snapshotterInjected.ts

```diff
@@ -432,10 +432,12 @@ export function frameSnapshotStreamer(snapshotStreamer: string, snapshotBinding:
       };
 
       let allOverridesAreRefs = true;
-      for (const sheet of this._allStyleSheetsWithUrlOverride) {
+      for (const sheet of this._staleStyleSheets) {
```

Comment:
I think we need to send all stylesheets that ever had an override every time, even when they are not stale (in this case it will be a number ref). Otherwise, the renderer will use the non-modified resource for this snapshot, instead of the last modified one.

Reply:
I won't claim I understand this part, but I don't change behavior here. Maybe you can follow up?